demo/src/demo.php

```diff
@@ -16,10 +16,15 @@
 
 $account = new Demo\Account();
 
-echo 'Deposit: 100' . PHP_EOL;
-$account->deposit(100);
+echo 'Deposit: 200' . PHP_EOL;
+$account->deposit(200);
 echo 'Current balance: ' . $account->getBalance();
 echo PHP_EOL;
 echo 'Withdraw: 50' . PHP_EOL;
 $account->withdraw(50);
-echo 'Current balance: ' . $account->getBalance();
+echo 'Current balance: ' . $account->getBalance(), PHP_EOL;
```

Comment:
Please use consistent concatenation using the `.` (dot)`
And also single quote please. Thanks 👍